internal/admin/dashboard/templates/page-failover.html

```
@@ -0,0 +1,121 @@
+
+    <div class="alert alert-warning" x-show="!failoverAvailable && !authError">
+        Failover feature is unavailable.
+    </div>
+    <div class="alert alert-warning" x-show="failoverError && !authError" x-text="failoverError"></div>
+    <div class="alert alert-success" x-show="failoverNotice && !failoverError" x-text="failoverNotice"></div>
+
+    <div class="table-toolbar">
+        <div class="table-toolbar-main">
+            <div class="filter-input-wrap">
+                <i data-lucide="search" class="filter-input-icon" aria-hidden="true"></i>
+                <input type="text" placeholder="Filter models..." aria-label="Filter models" x-model="modelFilter" class="filter-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)